x-pack/plugins/enterprise_search/public/applications/shared/role_mapping/__mocks__/elasticsearchUsers.ts

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export const elasticsearchUsers = [
```

Comment:
I think this file name should be snake cased `elasticsearch_users.ts` right?

Reply:
Yes, thanks.

Reply:
https://github.com/elastic/kibana/pull/102826/commits/0875f1f8ec561dfb8253457d5c828d4320397390